charter/01-00.md

```diff
@@ -1,6 +1,6 @@
 The WISH working group is chartered to specify a simple, extensible,
 HTTPS-based signaling protocol to establish one-way WebRTC-based audiovisual
```

Comment:
This sentence would also need to be amended to described more than a singular protocol.

Reply:
changed `protocol` by `set of protocols`, wdyt?